stable/artifactory-ha/values.yaml

```diff
@@ -1085,13 +1099,25 @@ nginx:
           proxy_pass       http://{{ include "artifactory-ha.fullname" . }}:{{ .Values.artifactory.externalPort }}/artifactory/$1;
         }
         proxy_pass          http://{{ include "artifactory-ha.fullname" . }}:{{ .Values.artifactory.externalPort }}/artifactory/;
+        {{- if .Values.nginx.service.ssloffload}}
+        proxy_set_header    X-Artifactory-Override-Base-Url $http_x_forwarded_proto://$host/artifactory;
+        {{- else }}
         proxy_set_header    X-Artifactory-Override-Base-Url $http_x_forwarded_proto://$host:$server_port/artifactory;
         proxy_set_header    X-Forwarded-Port  $server_port;
+        {{- end }}
         proxy_set_header    X-Forwarded-Proto $http_x_forwarded_proto;
         proxy_set_header    Host              $http_host;
         proxy_set_header    X-Forwarded-For   $proxy_add_x_forwarded_for;
       }
     }
+    {{- if .Values.nginx.service.ssloffload }}
```

Comment:
Why is this needed?

Reply:
@eldada @danielezer If SSL is terminated in AWS load balancer with ACM certificate, only the HTTP request goes to the service.